WSL/wsl-config.md

```diff
@@ -15,43 +15,34 @@ The [wsl.conf](#wslconf) and [.wslconfig](#wslconfig) files are used to configur
 
 You can configure the settings for your installed Linux distributions that will automatically be applied every time you launch WSL in two ways, by using:
 
-- **[.wslconfig](#wslconfig)** to configure settings **globally** across all installed distributions running on WSL 2.
-- **[wsl.conf](#wslconf)**  to configure settings **per-distribution** for Linux distros running on WSL 1 or WSL 2.
+- **[.wslconfig](#wslconfig)** to configure **global settings** across all installed distributions running on WSL 2.
+- **[wsl.conf](#wslconf)**  to configure **local settings** per-distribution for each Linux distribution running on WSL 1 or WSL 2.
 
-Both file types are used for configuring WSL settings, but the location where the file is stored, the scope of the configuration, and the version of WSL running your distribution all impact which file type to choose.
+Both file types are used for configuring WSL settings, but the location where the file is stored, the scope of the configuration, the type of options that can be configured, and the version of WSL running your distribution all impact which file type to choose.
 
-The version of WSL that you are running will impact the configuration settings. WSL 2 runs as a lightweight virtual machine (VM), so uses virtualization settings that allow you to control the amount of memory or processors used (which may be familiar if you use Hyper-V or VirtualBox).
+WSL 1 and WSL 2 run with different architecture and will impact the configuration settings. WSL 2 runs as a lightweight virtual machine (VM), so uses virtualization settings that allow you to control the amount of memory or processors used (which may be familiar if you use Hyper-V or VirtualBox). [Check which version of WSL you are running.](./install.md#check-which-version-of-wsl-you-are-running)
+
+## The 8 second rule for configuration changes
+
+You must wait until the subsystem running your Linux distribution completely stops running and restarts for configuration setting updates to appear. This typically takes about 8 seconds after closing ALL instances of the distribution shell.
+
+If you launch a distribution (e.g. Ubuntu), modify the configuration file, close the distribution, and then re-launch it, you might assume that your configuration changes have immediately gone into effect. This is not currently the case as the subsystem could still be running. You must wait for the subsystem to stop before relaunching in order to give enough time for your changes to be picked up. You can check to see whether your Linux distribution (shell) is still running after closing it by using PowerShell with the command: `wsl --list --running`. If no distributions are running, you will receive the response: "There are no running distributions." You can now restart the distribution to see your configuration updates applied.
```

Comment:
You can also use `wsl --terminate <distroName>` to terminate a running distro instantly.

Reply:
Added a note in the doc about this - thanks!